src/util/adaptSize.js

```diff
@@ -86,6 +86,15 @@ var adaptSize = (function() {
          */
         both: function($elements, minWidth, minHeight) {
             _resize($elements, { height: minHeight, width: minWidth });
+        },
+
+        /**
+         * Set height to auto on a set of elements
+         *
+         * @param {jQueryElements} $elements
+         */
+        resetHeight: function($elements) {
```

Comment:
```suggestion
        resetHeight($elements) {
```

Reply:
this will break inconsistency for this file